apps/native/app/src/messages/en.ts

```diff
@@ -594,6 +594,9 @@ export const en: TranslatedMessages = {
   'problem.offline.title': 'No internet connection',
   'problem.offline.message':
     'An error occurred while communicating with the service provider',
+  'problem.thirdParty.title': 'No connection',
```

Comment:
I feel like no connection means no internet. Isn't there a better sentence?

Failed to connect to service
Unable to reach service
Service unreachable

🤷‍♂️

Reply:
Yeah I agree, not a big fan of this, just used what the service portal is using 🤷‍♀️ Will discuss on sync today